### PR TITLE
moving non-extension admin action to admin controller

### PR DESCRIPTION
**Pull request by 2is10**
@leogrim

### CODE REVIEW

File: kifi-backend/modules/search/conf/search.routes [hunk 1]
```diff
@@ -1,10 +1,14 @@
 ##########################################
+# Search Admin
+##########################################
+GET         /search/instance      @com.keepit.controllers.search.SearchAdminController.instance()
```
Comment (2is10) [inline kifi-backend/modules/search/conf/search.routes]: we may want to put `/admin/` in the routes here, but I didn’t want to change the route, not knowing how it’s used today

Comment (ymatsuda) [inline kifi-backend/modules/search/conf/search.routes]: I agree. I put this for my convenience.

**Comment by ymatsuda** [comment]
lgtm

File: kifi-backend/modules/search/conf/search.routes [hunk 2]
```diff
@@ -1,10 +1,14 @@
 ##########################################
+# Search Admin
+##########################################
+GET         /admin/search/instance         @com.keepit.controllers.search.SearchAdminController.instance()
```
Comment (2is10) [inline kifi-backend/modules/search/conf/search.routes:4]: @ymatsuda wasn’t sure whether to use `/admin/search` or `/search/admin/` — feel free to switch it

**Pull request merged**